index.js

```diff
@@ -4,7 +4,7 @@ const Call = require('call');
 
 module.exports = (server, options, next) => {
   server.ext('onPreResponse', (request, reply) => {
-    if (request.response.isBoom && request.response.output.statusCode !== 404) {
+    if ((request.response.statusCode && request.response.statusCode !== 404) || (request.response.isBoom && request.response.output.statusCode !== 404)) {
```

Comment:
What's the scenario where you need to check if boom?

Reply:
when running the example and hitting a page that doesn't exist `request.response.statusCode` isn't defined. Instead they have the status code inside of that output object. 🤷‍♂️